x-pack/plugins/cases/public/components/case_view/translations.ts

```diff
@@ -137,30 +137,63 @@ export const DOES_NOT_EXIST_BUTTON = i18n.translate('xpack.cases.caseView.doesNo
 });
 
 export const TOTAL_ALERTS_METRIC = i18n.translate('xpack.cases.caseView.metrics.totalAlerts', {
-  defaultMessage: 'Total Alerts',
+  defaultMessage: 'Total alerts',
 });
 
 export const ASSOCIATED_USERS_METRIC = i18n.translate(
   'xpack.cases.caseView.metrics.associatedUsers',
   {
-    defaultMessage: 'Associated Users',
+    defaultMessage: 'Associated users',
   }
 );
 
 export const ASSOCIATED_HOSTS_METRIC = i18n.translate(
   'xpack.cases.caseView.metrics.associatedHosts',
   {
-    defaultMessage: 'Associated Hosts',
+    defaultMessage: 'Associated hosts',
   }
 );
 
 export const ISOLATED_HOSTS_METRIC = i18n.translate('xpack.cases.caseView.metrics.isolatedHosts', {
-  defaultMessage: 'Isolated Hosts',
+  defaultMessage: 'Isolated hosts',
 });
 
 export const TOTAL_CONNECTORS_METRIC = i18n.translate(
   'xpack.cases.caseView.metrics.totalConnectors',
   {
-    defaultMessage: 'Total Connectors',
+    defaultMessage: 'Total connectors',
   }
 );
+
+export const CASE_CREATED = i18n.translate('xpack.cases.caseView.metrics.lifespan.caseCreated', {
+  defaultMessage: 'Case created',
+});
+
+export const CASE_IN_PROGRESS_DURATION = i18n.translate(
+  'xpack.cases.caseView.metrics.lifespan.inProgressDuration',
+  {
+    defaultMessage: 'Case in progress duration',
+  }
+);
+
+export const CASE_OPEN_DURATION = i18n.translate(
+  'xpack.cases.caseView.metrics.lifespan.openDuration',
+  {
+    defaultMessage: 'Case open duration',
+  }
+);
+
+export const CASE_OPEN_TO_CLOSE_DURATION = i18n.translate(
+  'xpack.cases.caseView.metrics.lifespan.openToCloseDuration',
+  {
+    defaultMessage: 'Duration from case creation to close',
+  }
+);
+
+export const CASE_REOPENED = i18n.translate('xpack.cases.caseView.metrics.lifespan.reopened', {
+  defaultMessage: '(reopened)',
+});
+
+export const CASE_REOPENED_ON = i18n.translate('xpack.cases.caseView.metrics.lifespan.reopenedOn', {
+  defaultMessage: 'Case was reopened on ',
```

Comment:
It might also be a bit shorter.
```suggestion
  defaultMessage: 'Reopened on ',
```